sys/logging/logging.h

```diff
@@ -28,7 +28,7 @@ typedef enum { LOG_TRACE, LOG_DEBUG, LOG_INFO, LOG_WARN, LOG_ERROR, LOG_FATAL, L
 
 #define RETURN_IF_ERROR_CODE(_ret)         \
   do {                                     \
-    errCode = _ret;                        \
+    error_code_t errCode = _ret;                        \
```

Comment:
There is no reason to modify this. This will create way to make errCode variables that will eat up the stack size. Instead if a function uses this macro, put a errCode at the top of the function scope and it will work properly